src/app/components/InputWindow.tsx

```
@@ -6,37 +6,60 @@ interface InputWindowProps {
+
+	const handleFocusOut = () => {
+		onBlur?.();
+		if (value === '') {
+			setTypeStatus('empty');
+		}
+	};
+
 	// 부모로부터 error 전달 받을 경우 error state 설정
 	useEffect(() => {
 		if (isError) {
+			console.log('자식: error');
```

ESLint

[error] Unexpected console statement.

(no-console)